proto/src/determined/api/v1/config_policies.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+
+package determined.api.v1;
+option go_package = "github.com/determined-ai/determined/proto/pkg/apiv1";
+
+import "google/protobuf/struct.proto";
+
+
+
+// PutWorkspaceConfigPoliciesRequest is a complete replacement of config policies for the workspace and workload type.
+message PutWorkspaceConfigPoliciesRequest {
+  int32 workspace_id = 1;
+  string workload_type = 2;
```

Comment:
We can't use an enum as a url path parameter. We can use an int or string and then map it to an enum for the backend. Query parameters can be enums, but that doesn't work for our `GET` or `DELETE` APIs.

Reply:
Good to know! Huh, is this a gRPC Gateway limitation? Maybe this is worth calling out in the comment for this field.

Reply:
Yeah, pretty much. Enums should be defined as a query parameter, rather than a path parameter, in the proto file but gRPC Gateway doesn't support `query_params` for DELETE requests (to the best of my understanding and recollection).